packages/cloud_functions/cloud_functions/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: cloud_functions
 description: Flutter plugin for Cloud Functions.
-version: 0.4.1+8
+version: 0.4.1+9
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/cloud_functions/cloud_functions
 
 flutter:
```

Comment:
also add a `web:` section with a `default_package:` like here: https://github.com/flutter/plugins/blob/master/packages/url_launcher/url_launcher/pubspec.yaml#L16

Reply:
Ah, so that's how that happens. Okay, will do.